wear/src/main/java/io/homeassistant/companion/android/sensors/HealthServicesSensorManager.kt

```diff
@@ -62,22 +97,36 @@ class HealthServicesSensorManager : SensorManager {
 
     override fun requestSensorUpdate(context: Context) {
         latestContext = context
-        updateUserActivityState()
+        updateHealthServices()
     }
 
-    private fun updateUserActivityState() {
-        if (!isEnabled(latestContext, userActivityState.id)) {
-            passiveMonitoringClient?.clearPassiveListenerCallbackAsync()
-            callBackRegistered = false
+    private fun updateHealthServices() {
+        val activityState = isEnabled(latestContext, userActivityState.id)
+
+        if (!activityState && !isEnabled(latestContext, dailyFloors.id)) {

```

Comment:
Daily floors enabled is also used later to determine the data types to be included, so for consistency maybe put it in a variable like activity state enabled?

Reply:
Good suggestion :) will make it easier for others to add sensors later